backend/Origam.Workflow/Service Agents/SmsServiceAgent.cs

```diff
@@ -0,0 +1,70 @@
+﻿#region license
+/*
+Copyright 2005 - 2021 Advantage Solutions, s. r. o.
+
+This file is part of ORIGAM (http://www.origam.org).
+
+ORIGAM is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+ORIGAM is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with ORIGAM. If not, see <http://www.gnu.org/licenses/>.
+*/
+#endregion
+
+using Origam.Sms;
+using System;
+
+namespace Origam.Workflow
+{
+    public class SmsServiceAdapter : AbstractServiceAgent
+    {
+        private object _result;
+        public override object Result
+        {
+            get
+            {
+                var temp = _result;
+                _result = null;
+				
+                return temp;
+            }
+        }
+        public override void Run()
+        {
+            switch (MethodName)
+            {
+                case "SendSms":
+                    _result = CreateSmsService().SendSms(
+                        GetParameter<string>("from"),
+                        GetParameter<string>("to"),
+                        GetParameter<string>("to"));
```

Comment:
This should probably be `body`, not `to`?

Reply:
Yes, I was talking about it at a recent call, weird I committed older. Will Fix